documentation/docs/10-getting-started/10-introduction.md

```diff
@@ -1,3 +1,4 @@
+[Export-dadd744f-3cce-45c4-bccf-30dae7e8ed0e.zip](https://github.com/sveltejs/kit/files/11125441/Export-dadd744f-3cce-45c4-bccf-30dae7e8ed0e.zip)
```

Comment:
This looks dangerous. 

![image](https://user-images.githubusercontent.com/38754555/229226083-5a55705b-990a-484e-90ed-0bea6656f5ad.png)

This what the zip content has. Malicious PR. Plus his contributions are set to private for whatever reason.